src/actions/guides/frontend/html_forms.cr

```diff
@@ -0,0 +1,533 @@
+class Guides::Frontend::HtmlForms < GuideAction
+  guide_route "/frontend/html-forms"
+
+  def self.title
+    "HTML Forms"
+  end
+
+  def markdown : String
+    <<-MD
+    ### Working with Lucky's HTML builder
+
+    All HTML tags have an associated method based on the name of the tag, with the exception of a few.
+    (e.g. `<form></form>` - `form()`, `<button></button>` - `button()`, etc...)
+
+    These are the only HTML tags with an alias name method.
+
+    * `<select></select>` - `select_tag`
+    * `<p></p>` - `para`
+
+    > These are overridden as to not clash with built-in Crystal methods.
```

Comment:
This seems like a general overview which feels odd to have duplicated from the rendering html guide. I'm thinking the first part of this paragraph could be something like:

> You can generate form tags like `<form>`, `<input>`, etc. using [methods like `form` and `input`]#{link to rendering html}}, but the recommended way is to use Lucky's form helpers.

Then lead with the rest of what you have